builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -292,14 +292,16 @@ node() {
 						events.sendCompletedEvent('CREATE_ASSET', null, utilModule.generateAssetMap("aws", swaggerDocUrl, "swagger_url", config), environment_logical_id);
 
 						def svc_status = "Your service endpoint for ${current_environment} environment: ${endpointUrl} \n\nView and test your API here: ${configLoader.JAZZ.SWAGGER.EDITOR_URL}${swaggerDocUrl}"
-						send_status_email(config, 'COMPLETED', svc_status)
+
+            echo "$svc_status"
```

Comment:
do we need this?

Reply:
we cant identify the API end point from the console

Reply:
Removed the echo